docs/user/index.rst

```diff
@@ -0,0 +1,120 @@
+######################
+Using Aldryn Reversion
+######################
+
+After you set up and configured your models (they are registered for revisions
+support) you can start using ``aldryn-reversion`` end-user features.
+
+************************
+Restoring object version
+************************
+
+Most likely you already know that django keeps track of object changes also
+known as history, it is available on object admin edit form on the top of the
+page. With ``aldryn-reversion`` that history has more features.
+As soon as object is created - 'Initial version' is created and treated as
+restore point.
+
+Each record in history has a date which is now a link to restore revision
+form.
+When you need to restore any object version you just need to click on desired
+date, check what will be changed  and either restore that revision or go back
+to object history.
+
+.. note::
+    Be aware that in current implementation revision is restored as is.
+    That means that any related object that is also stored in this revision
+    would be restored at the point at which that object was stored.
+    This is applicable to whole relation tree (if follow was configured).
+
+As soo as you restore object version the object would be at the version at
+which it was kept and you will be redirected to object edit form so that you
+can change object if you need to.
+
+You might also want to change related object (if applicable) since they also
+will be affected.
+
+.. note::
+   If :ref:`follow` was not configured properly and related object was not
+   stored in selected revision - you might end up with getting an
+   ``IntegrityError``. In this case you first need to restore related object
+   that was deleted.
+
+
+*************************
+Recovering deleted object
+*************************
+
+Second most powerful feature of ``aldryn-reversion`` is to recover deleted
+objects.
+You may find it on model's change list page after clicking on the
+``Recover deleted`` button on top of that page.
+
+If there are deleted objects (which do not exist in database but there is an
```

Comment:
"If there are any deleted objects which do not exist in the database but for which there is a version stored by reversions, they will be listed on the recovery page."